### ISSUE TITLE
Unrecognized option after installation

### ISSUE THREAD

**Issue by fbara** [Xentrk/x3mRouting#43]
I have an Asus RT-AC86U running Merlin v386.2_2.  I have 3 OpenVPN clients setup using the info provided by ExpressVPN.  These clients have been working without a problem for weeks.

I installed x3MRouting following the instructions on this site for option 2 `Install OpenVPN Client GUI, OpenVPN Event & x3mRouting.sh Script`.  I didn’t receive any errors during the installation process and I can see ‘x3MRouting Open VPN Client Settings’ title on the OpenVPN Client tab of the router.

After the installation, I immediately received an ‘Error - check configuration’ on in the Service State of each VPN client.  None of the clients could connect.  I have this error in the log:
`ovpn-client1[4617]: Options error: Unrecognized option or missing or extra parameter(s) in config.ovpn:19: verify-x509-name (2.5.1)`

I checked Custom Configuration for the offending line in ovpn-client1 but didn’t see verify-x509-name:
```fast-io
remote-random
pull
tls-client
ns-cert-type server
route-method exe
route-delay 2
tun-mtu 1500
fragment 1300
mssfix 1200
cipher AES-256-CBC
keysize 256
sndbuf 524288
rcvbuf 524288
```
I added verify-x509-name but still received the same error.  

I removed the OpenVPN client completely and tried re-added the .ovpn file but the error returned.

**Comment by fbara** [comment]
I manually changed the .ovpn file from ExpressVPN and commented out `verify-x509-name`.  I deleted all vpn configs from the router, rebooted, and added the modified .ovpn back.  It started up without any problems.

I modified 2 additional .ovpn files the same and they also loaded and started without a problem.  I reloaded x3MRouting, option 2, and everything looks good now.

I’m not smart enough to know why this worked but it did.  Closing this issue.
